travis/travis_install_nightly

```diff
@@ -32,6 +32,11 @@ ln -s `which nodejs` $HOME/maintainer-quality-tools/travis/node
 npm install -g less less-plugin-clean-css
 
 
+# Update PhantomJS (v10 compat)
+npm install --prefix ${TRAVIS_BUILD_DIR} phantomjs-prebuilt@2.1.14
```

Comment:
I mean, maybe odoo 8.0 requires a old version of phantomjs or other odoo version require a particular phantomjs version where we need use a environment variable to assign it

If the version is not assigned then use the default OS version